src/index.js

```diff
@@ -47,14 +47,12 @@ window.addEventListener('load', async() => {
   if (window.ethereum) {
     window.web3 = new constants.Web3(window.ethereum)
     try {
-      await window.ethereum.enable()
+      await window.ethereum.request({method: `eth_requestAccounts`})
```

Comment:
[Metamask docs](https://docs.metamask.io/guide/getting-started.html#connecting-to-metamask):
>You should never initiate a connection request on page load.

Though, it should be address as a separate issue. Let's bring it back ASAP.